core-errors/src/lib.rs

```diff
@@ -22,10 +22,14 @@ use codec::{Decode, Encode};
 use core::fmt;
```

Comment:
Left the comment in gcore about codec. better to provide here feature "codec" to impl `Encode + Decode + TypeInfo` if only feature appeared

Reply:
> better to provide here feature "codec" to impl Encode + Decode + TypeInfo if only feature appeared

What's the matter if `gr_error` always need `Decode`?

Reply:
Because gcore as minimal lib as it could be.
codec brings a lot of bloat for final binary size